snowplow_tracker/events/event.py

```diff
@@ -0,0 +1,73 @@
+# """
+#     event.py
+
+#     Copyright (c) 2013-2023 Snowplow Analytics Ltd. All rights reserved.
+
+#     This program is licensed to you under the Apache License Version 2.0,
+#     and you may not use this file except in compliance with the Apache License
+#     Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+#     http://www.apache.org/licenses/LICENSE-2.0.
+
+#     Unless required by applicable law or agreed to in writing,
+#     software distributed under the Apache License Version 2.0 is distributed on
+#     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#     express or implied. See the Apache License Version 2.0 for the specific
+#     language governing permissions and limitations there under.
+# """
+
+from typing import Optional, List
+from snowplow_tracker import payload
+from snowplow_tracker import subject as _subject
+
+from snowplow_tracker.self_describing_json import SelfDescribingJson
+
+from snowplow_tracker.constants import CONTEXT_SCHEMA
+from snowplow_tracker.typing import JsonEncoderFunction, PayloadDict
+
+
+class Event(object):
+    def __init__(self, dict_: Optional[PayloadDict] = None) -> None:
+        """
+        Constructor
+        """
+        self.pb = payload.Payload(dict_=dict_)
```

Comment:
Nit: I think it would be nicer to rename `pb` to `payload`. It's a bit more self-explanatory to use that full name.